p2p/wdrr_queue.go

```diff
@@ -31,7 +31,7 @@ var _ queue = (*wdrrScheduler)(nil)
 // - quantum: The number of bytes that is added to the deficit counter of the
 //   flow in each round. The flow can send at most quantum bytes at a time. Each
 //   flow has its own unique quantum, which gives the queue its weighted nature.
-//   A higher quantum corresponds to a higher weight/priority. The quantum is
+//   A highzer quantum corresponds to a higher weight/priority. The quantum is
```

Comment:
que?

Reply:
![](https://media.giphy.com/media/XeXzWgD6P4LG8/giphy.gif)

Reply:
I think I did that thing where I think I'm on one tab when I'm actually on some other one just typing away

Reply:
Thanks for catching me in any case :)